.github/workflows/review-release-notes.yml

```diff
@@ -12,9 +15,13 @@ jobs:
           fetch-depth: 2
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v18.7
+        uses: tj-actions/changed-files@v35.1.0
         with:
-          separator: ';'
+          separator: ${{ env.CHANGED_FILES_DELIMITER }}
+          files: |
+            Packs/**/ReleaseNotes/**
```

Comment:
Nice addition :) 
@DinaMeylakh - please confirm that we are not missing anything by doing that.

Reply:
From what I see in https://github.com/demisto/content/blob/CIAC-4970-run-release-notes-check-only-if-xsoar-supported-pa/Utils/github_workflow_scripts/run_docs_review.py#L27, we're specifying that only release notes will be processed through the doc review:

```python
doc_reviewer = DocReviewer(file_paths=files_for_review,
                               release_notes_only=True, # <-----------
                               known_words_file_paths=['Tests/known_words.txt'],
                               load_known_words_from_pack=True,
                               no_camel_case=True)
```

So I believe there's no need to have any other changed files (besides release notes) in this context.